cmd/trayscale/systray.go

```diff
@@ -1,6 +1,43 @@
 package main
 
-import "fyne.io/systray"
+import (
+	"fyne.io/systray"
+)
+
+type tray struct {
+	showItem *systray.MenuItem
+	quitItem *systray.MenuItem
+}
+
+func initTray(online bool) *tray {
+	systray.SetIcon(statusIcon(online))
+	systray.SetTitle("Trayscale")
+
+	showWindow := systray.AddMenuItem("Show", "")
+	systray.AddSeparator()
+	quit := systray.AddMenuItem("Quit", "")
+
+	return &tray{
+		showItem: showWindow,
+		quitItem: quit,
+	}
+}
+
+func (t *tray) NotfiyQuit() <-chan struct{} {
```

Comment:
I'm not thrilled with the `Notify` naming convention. How about just `QuitChan()`?

Reply:
I've got no strong opinions here. My initial design was influenced by the APIs in `rabbitmq/amqp091-go`. 
This is fine too.